oci-library/oci-hol/java-management/set-up-logrotate/set-up-logrotate.md

```diff
@@ -0,0 +1,138 @@
+# Set up logrotate
+
+## Introduction
+
+This lab will guide you in setting up the logrotate utility on your host machine to periodically rotate the `usagetracker.log` generated by Java Usage Tracker.
+
+Although the size of the Java Usage Tracker log file is small, you should consider periodically truncate, compress, archive, or delete the log file. This is because the Java Usage Tracker does not check for available disk space or perform these administrative tasks on the log file as it incrementally adds records to it. 
```

Comment:
Tenses: Although the size of the Java Usage Tracker log file is small, you should consider periodically truncating, compressing, archiving, or deleting the log file.